Misc/NEWS.d/next/Library/2018-12-09-17-04-15.bpo-17185.SfSCJF.rst

```diff
@@ -0,0 +1,2 @@
+Set ``__signature__`` on mock for `inspect` to get signature. Fix signature
+check for partial and partialmethods. Patch by Karthikeyan Singaravelan.
```

Comment:
Please, cross reference with `functools.partial` and `functools.partialmethod` using rst markup.